app/models/user.rb

```diff
@@ -1,11 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
```

Comment:
- [ ] Dear Team, we need to enable `:confirmable` method to enable the application to use the confirmation email, as stated in the project requirement the application is expected to ask for confirmation of email.